packages/Meshfree/src/DTK_SplineOperator_def.hpp

```diff
@@ -418,34 +250,41 @@ void SplineOperator<DeviceType, CompactlySupportedRadialBasisFunction,
     DTK_REQUIRE( source_values.extent( 0 ) == _n_source_points );
     DTK_REQUIRE( target_values.extent( 0 ) == target_offset.extent( 0 ) - 1 );
 
+    // Step 2: Construct vectors
+    auto source = Teuchos::rcp( new Vector( S->getDomainMap(), 1 ) );
+    auto destination = Teuchos::rcp( new Vector( N->getRangeMap(), 1 ) );
+
     // Retrieve values for all source points
     source_values = Details::NearestNeighborOperatorImpl<DeviceType>::fetch(
-        _comm, _ranks, _indices, source_values );
+        _comm, _ranks, indices, source_values );
```

Comment:
I may have prematurely got rid of `_ranks` and `_indices`. May need to restore them. But before those were coming from the N matrix, which is probably not right.

Reply:
Really? Wasn't that `target_ranks` and `target_indices`?